docs/release-notes/outdated-checkout-endpoint-deprecation.md

```diff
@@ -4,6 +4,7 @@ title: "Outdated Checkout endpoint deprecation"
 createdAt: 2023-01-16T14:01:00.322Z
 hidden: false
 type: "deprecated"
+excerpt: The Outdated Checkout endpoint is an old endpoint that will be discontinued after June 25th, 2023.
```

Comment:
```suggestion
excerpt: The Outdated Checkout endpoint will be discontinued after June 25th, 2023.
```

Reply:
Fica repetitivo, com o "outdated" no início

Reply:
Outdated neste caso foi o nome que demos para identificar a API.